sebs.py

```diff
@@ -121,8 +122,8 @@ def parse_common_params(
     logging_filename = os.path.abspath(os.path.join(output_dir, output_file))
 
     sebs_client = sebs.SeBS(cache, output_dir, verbose, logging_filename)
-    output_dir = sebs.utils.create_output(output_dir, preserve_out, verbose)
-    sebs_client.logging.info("Created experiment output at {}".format(output_dir))
+    output_dir = sebs.utils.create_output(output_dir, preserve_out, verbose)    
+    ColoredPrinter.print(Colors.STATUS, sebs_client.logging, "Created experiment output at {}".format(output_dir))
```

Comment:
One possible implementation is to add to the logging base a `@property` function called `logging` that would return your colored wrapper instead of the original logger. This should seamlessly affect the entire project.

Reply:
I have implemented all wrappers for logging, critical, error, warning, info and debug. Each of them are mapped to the corresponding appropriate colors.